twilio-sms/twilio-sms.lua

```diff
@@ -0,0 +1,117 @@
+--[[
+
+# [TWILIO MESSAGING API](https://www.twilio.com/docs/api/messaging) connector.
+
+### Authorization
+
+Authorization use [Basic authentication](https://www.twilio.com).
```

Comment:
Link should point to https://www.twilio.com/docs/api/messaging#messaging-api-authentication